src/core/js/scriptLoader.js

```diff
@@ -30,7 +30,8 @@
                 a11y: 'libraries/jquery.a11y',
                 scrollTo: 'libraries/scrollTo',
                 bowser: 'libraries/bowser',
-                'enum': 'libraries/enum'
+                enum: 'libraries/enum',
+                jqueryMobile: 'libraries/jquery.mobile.custom'
```

Comment:
I guess the reason I started using swipe interactions for mobile is it offers the user a way of interacting that's intuitive and something they're used to doing on other sites/apps. Allowing swiping with the mouse does the opposite ie it's counter-intuitive and fiddly so there doesn't seem much point. Having said that, most users would therefore not even be aware that they were able to do it, but I guess part of me would like it not to be there if it's not going to be used. I'm happy to add the condition if the consensus is that's something we want to do. Equally happy to continue as is :)